geopandas/geoseries.py

```diff
@@ -132,9 +132,9 @@ class GeoSeries(GeoPandasBase, Series):
     - Lat[north]: Geodetic latitude (degree)
     - Lon[east]: Geodetic longitude (degree)
     Area of Use:
-    - name: World
+    - name: World.
     - bounds: (-180.0, -90.0, 180.0, 90.0)
-    Datum: World Geodetic System 1984
+    Datum: World Geodetic System 1984 ensemble
```

Comment:
This is only the case with PROJ 8+. Any way to version check this with the tests?

Reply:
Not that I am aware of. I think it is fine, we check docstrings in one env only for reasons like this. So as long as it is in sync with this env, we're fine I think. It is a minor difference anyway.